include/ck_tile/ops/gemm/pipeline/gemm_pipeline_ag_bg_cr_comp_v4.hpp

```diff
@@ -0,0 +1,551 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2025, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
The year convention is as follow: \<year-of-creation\>[-\<year-of-last-modification-if-different-from-year-of-creation\>]
```suggestion
// Copyright (c) 2025, Advanced Micro Devices, Inc. All rights reserved.
```